model/src/main/java/org/cbioportal/model/MolecularProfileSampleCount.java

```diff
@@ -9,6 +9,8 @@
 	private Integer numberOfCNAProfiledSamples;
```

Comment:
For this work, we have chosen to keep the fusionProfiledSample list parallel to the mutationProfiledSample list ... in other words there can be samples which were profiled for fusions/structural variants and in which no events were found (for any gene). CNA events have a representation in the portal database formatted not as single records representing events (such as mutations and fusions) but as event call vectors ... lists of numbers showing the collection of event (or non-event) codes for every sample in the profile ... one vector per profiled gene. Because of this, data is present inherently for all samples in the profile (even if the data records a non-event). This means that we do not have the same difficulties distinguishing the samples which were not profiled from the samples which were profiled but did not appear in the list of events at all.
However, we have considered moving away from this representation and moving towards a representation which only contains events (and not codes for non-events). If we make any modifications to the CNA representation, we should consider using the cases_cna case_list as the authority for determining which samples were profiled in a way which would detect CNA. So we should look for the first opportunity to make this concept parallel across all molecular_data_profile datatypes.